sapi/cli/php_cli_server.c

```diff
@@ -2485,14 +2485,11 @@ static int php_cli_server_ctor(php_cli_server *server, const char *addr, const c
 		server->router_len = 0;
 	}
 
-	if (php_cli_server_mime_type_ctor(server, mime_type_map) == FAILURE) {
-		retval = FAILURE;
-		goto out;
-	}
+	php_cli_server_mime_type_ctor(server, mime_type_map);
 
 	server->is_running = 1;
 out:
-	if (retval != SUCCESS) {
```

Comment:
Personally I like the defensive/whitelist approach of `!= SUCCESS`, and I can't find an objective argument in favor of changing this to `== FAILURE`. I feel that it's mostly a matter of taste :)

I'm not against this change, but unless we have a clear coding standard regarding this, we risk changing the same lines back and forth between different styles.

Reply:
The main reason I do these change is if I see ``!= SUCCESS`` I don't know if the call can only return ``zend_result`` or some other value, as I have already seen that in the code base where it *looks* like the called function only returns ``zend_return`` but does not (e.g. returns ``1``) so it is more a mental marker for me. But I'm not really attached to the equality.

Reply:
I generally agree with @arnaud-lb. I don't prefer one over the other, and I'm happy making one the default. But I fear that unless we do have a standard people keep changing things like this back and forth for no good reason.